### ISSUE TITLE
[Feature Request] hide procs / focus mode

### ISSUE THREAD

**Issue by LLBlumire** [pvolok/mprocs#71]
We can already hide the keybinds panel, but it would be great to be able to hide everything (on a keypress) and only see the selected process terminal.

**Comment by pvolok** [comment]
There is a zoom mode that you can enable by pressing 'z'. Is this what you are looking for?

**Comment by LLBlumire** [comment]
Ah, that's exactly it, it might be good to clarify the README to explain what that does, as it seems to only ever be listed as "Zoom into terminal window", perhaps "Zoom into terminal window, hiding all other UI elements"

**Comment by pvolok** [comment]
We can make the terminal borderless in the zoom mode. And add `hide_zoom_info` to config to allow hiding the top panel.

**Comment by LLBlumire** [comment]
Yeah I think that would be ideal! Some people might like the borders to remind them they are in mprocs, maybe adding `hide_zoom_info` and `hide_zoom_borders` as separate toggles?

**Comment by pvolok** [comment]
Well, that might be useful. But there is a top panel to remind you of the mode. So I'd like to avoid adding more config options unless they are really needed.

Regarding the implementation, I'm redesigning some internals of mprocs, including the layout. I will include this feature into that redesign.
